ydb/core/driver_lib/cli_utils/cli_cmds_tenant.cpp

```diff
@@ -627,6 +675,15 @@ class TClientCommandTenantPools : public TClientCommandTree {
     }
 };
 
+class TClientCommandTenantQuotas : public TClientCommandTree {
+public:
+    TClientCommandTenantQuotas()
+        : TClientCommandTree("quotas", {}, "")
```

Comment:
You forgot to write the command description as the third argument. It's needed for displaying in --help.

Reply:
I did the same here as in [`units`](https://github.com/ydb-platform/ydb/blob/f1b17cefba75c57c4c3f804599db55a19318a00a/ydb/core/driver_lib/cli_utils/cli_cmds_tenant.cpp#L660) and [`pools`](https://github.com/ydb-platform/ydb/blob/f1b17cefba75c57c4c3f804599db55a19318a00a/ydb/core/driver_lib/cli_utils/cli_cmds_tenant.cpp#L672) commands:

Is it better to add a description?

Reply:
But [create](https://github.com/ydb-platform/ydb/blob/main/ydb/core/driver_lib/cli_utils/cli_cmds_tenant.cpp#L311), [remove](https://github.com/ydb-platform/ydb/blob/main/ydb/core/driver_lib/cli_utils/cli_cmds_tenant.cpp#L382) and [status](https://github.com/ydb-platform/ydb/blob/main/ydb/core/driver_lib/cli_utils/cli_cmds_tenant.cpp#L239) commands have description. It is better to write description in quotas cmd